examples/fleet/kuard/README.md

```diff
@@ -0,0 +1,116 @@
+# Example: Creating a multi-cluster service that exposes endpoints from a number of Kubernetes clusters
+
+This directory contains the instructions on how to create a multi-cluster service that exposes endpoints on fleet.
+
+The application is using [kuard](https://github.com/kubernetes-up-and-running/kuard) as the service to demonstrate the responding cluster/pod.
+
+## Before you begin
+* Install [kubectl](https://kubernetes.io/docs/tasks/tools/).
+* Create a fleet with two members: `aks-member-blue`and `aks-member-green` using [Azure CNI networking](https://review.learn.microsoft.com/en-us/azure/aks/configure-azure-cni).
+* Member clusters reside on the same virtual network, or they reside on different virtual networks that are [peer-linked](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-peering-overview), i.e. Pods from different clusters can communicate directly with each other using their assigned Pod IP addresses.

```

Comment:
Shorten this?

Member clusters reside on the same virtual network or [peered virtual networks](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-peering-overview) such that pods from different clusters can communicate directly with each other using pod IPs.